distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/XSearch.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.itests.common;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
+
+import org.w3c.dom.Document;
+
+/**
+ * Simpler helper class for XPath and DOM related functions.
+ * The purpose is to reduce the amount of duplicated code used in the test classes.
+ * <p>
+ * This class is not thread-safe. It not optimized-- it creates new factories and
+ * builders for every invocation. It is not suitable for use in production code.
+ * <p>
+ * This class does not catch errors. It passes errors up the call chain to provide developers with
+ * the most information possible when debugging test cases.
+ * <p>
+ * Unlike ddf.util.XPathHelper, this class can ignore XML namespaces.
+ * <p>
+ * To format (pretty print) XML, use the ddf.util.XPathHelper or
+ * org.codice.ddf.platform.util.XMLUtils classes.
+ * <p>
+ * For XML related test assertions, use an org.custommonkey.xmlunit.XMLAssert methods like
+ * assertXpathEvaluatesTo().
+ * <p>
+ * For XML related test assertions in REST Assured responses,
+ * use the Hamcrest matcher method hasXpath().
+ */
+public class XSearch {
+
+    /**
+     * Create an DOM from a string representation of an XML document.
```

Comment:
I reviewed the public methods. I found that two @return statements were incomplete and I fixed them. I didn't see any missing argument or method descriptions. I did not document the `getXpath` method because it `protected` and it is very simple.